docs/install.md

```diff
@@ -14,7 +14,7 @@
 - [Standalone Releases](#standalone-releases)
 - [Docker](#docker)
 - [helm](#helm)
-- [App Engines (Azure, Heroku)](#app-engines)
+- [App Engines (Azure, Heroku)](#app-engines-azure-heroku)
```

Comment:
Not sure what happened with that title link, but this should fix. automatically added by `yarn fmt`